tdp/cli/commands/conftest.py

```diff
@@ -31,3 +35,15 @@ def collection_path(tmp_path_factory: pytest.TempPathFactory) -> Path:
 @pytest.fixture
 def vars(tmp_path_factory: pytest.TempPathFactory) -> Path:
     return tmp_path_factory.mktemp("collection")
+
+
+@pytest.fixture
+def create_cluster_variables(collection_path, vars) -> ClusterVariables:
```

Comment:
Other fixtures do not have a verb, you should rename it `cluster_variables`.
```suggestion
def cluster_variables(collection_path, vars) -> ClusterVariables:
```

Reply:
done